### PR TITLE
Validation tests maintenance

### LINKED ISSUE #1162: Unsupported methods in Validation tests
Some validation tests are not properly tested because of a JS exception.
- Test should fail with JS exception
- Unsupported methods should be ...well, supported

```
Running Outline
TypeError: Object doesn't support property or method '_addPendingData'


Running GUI3D SpherePanel
TypeError: Object doesn't support property or method 'flat'
```

These errors are visible in the nightly build logs.

### PR DESCRIPTION
**Pull request by CedricGuillemet**
- update PG versions for missing methods (in sync with bjs config.json)
- Validation test display a message when no pixel difference
- Update Babylon.js

fixes #1162

### CODE REVIEW

**Comment by CedricGuillemet** [comment]
Looks like V8 is not collecting destroyed framebuffer fast enough. And I don't want to have a specific polyfill for Chakra.

File: Apps/ValidationTests/Scripts/validation_native.js [hunk 1]
```diff
@@ -38,6 +38,8 @@ function compare(test, renderData, referenceImage, threshold, errorRatio) {
 
     if (differencesCount) {
         console.log("Pixel difference: " + differencesCount + " pixels.");
+    } else {
+        console.log("No pixel difference !!!");
```
Comment (PolygonalSun) [inline Apps/ValidationTests/Scripts/validation_native.js:42]: Nit: Remove space between difference and !!!

**Review by PolygonalSun — APPROVED** [review]
LGTM

File: Apps/ValidationTests/Scripts/validation_native.js (hunk 1 above)
Comment (CedricGuillemet) [reply Apps/ValidationTests/Scripts/validation_native.js]: Done

**Pull request merged**